### PR TITLE
Support :production in Plaid::Client#env=

### PR DESCRIPTION
**Pull request by be9**
`:api` was replaced with `:production`.

### CODE REVIEW

**Comment by be9** [comment]
@rbuckheit please have a look :) this one is really minimal.

File: lib/plaid/client.rb [hunk 1]
```diff
@@ -14,33 +14,35 @@ class Client
 
     # Public: Set Plaid environment to use.
     #
-    # env - The Symbol (:tartan, :api), or a full String URL like
+    # env - The Symbol (:tartan, :production), or a full String URL like
     #       'https://tartan.plaid.com'.
     def env=(env)
       case env
       when :tartan, :api
```
Comment (rbuckheit) [inline lib/plaid/client.rb]: why do we have both `:api` and `:production`? It looks like they point to the same thing.

Comment (rbuckheit) [inline lib/plaid/client.rb]: i think we can remove `:api` in the when case above if it's no longer supported.

**Comment by be9** [comment]
@rbuckheit it's only `:production` now.

**Comment by rbuckheit** [comment]
:fish: lgtm, thanks @be9 !

**Pull request merged**